website/pages/docs/nia/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+page_title: Network Infrastructure Automation <sup>Tech Preview</sup>
+sidebar_title: Network Infrastructure Automation <sup>Tech Preview</sup>
+description: >-
+  Network Infrastructure Automation (NIA) relies on a declarative, workflow and service driven network automation architecture. Consul-Terraform-Sync is a daemon based architecture that utilizes Terraform as the underlying automation tool and leverages the Terraform provider ecosystem to drive relevant change to the network infrastructure. This usage of the Terraform provider in conjunction with Consul allows for Consul to act as the core data source to inform updates to the state of the infrastructure.
+---
+
+# Network Infrastructure Automation <sup>Tech Preview</sup>
+
+Network Infrastructure Automation (NIA) enables dynamic updates to network infrastructure devices triggered by service changes. Consul-Terraform-Sync utilizes Consul as a data source that contains networking information about services and monitors those services. Terraform is used as the underlying automation tool and leverages the Terraform provider ecosystem to drive relevant change to the network infrastructure.
+
+Consul-Terraform-Sync executes one or more automation tasks with the most recent service variable values from the Consul service catalog. Each task consists of a runbook automation written as a Consul-Terraform-Sync compatible Terraform module using resources and data sources for the underlying network infrastructure. The `consul-terraform-sync` daemon runs on the same node as a Consul agent.
+
+## Use Cases
+
+**Application teams must wait for manual changes in the network to release, scale up/down and re-deploy their applications.** This creates a bottleneck, especially in frequent workflows related to scaling up/down the application, breaking the DevOps goal of self-service enablement. Consul-Terraform-Sync automates that process which decreases the time in which those changes are made and decreases the possibility of human error.
+
+**Networking and security teams cannot scale processes to the speed and changes needed.** Many network and security teams are struggling to keep up with the rate of change because they are using manual approaches. Even teams which have implemented a degree of network automation are struggling to keep up with the change due to a lack of accurate, real-time source of data to trigger and drive their network automation workflows. Consul-Terraform-Sync runs in near real-time to keep up with the rate of change.
```

Comment:
Would suggest a more direct style of writing here, something like  "Manual approaches don't scale well, causing backlogs in network and security teams. Even in organizations that have some amount of automation (such as scripting), there is a need for an accurate, real-time source of data to trigger and drive their network automation workflows. Consul-Terraform-Sync runs in near real-time to keep up with the rate of change"

Reply:
👍 Thank you!